client/tokenized-payment-request/transformers/wc-to-stripe.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const transformCartDataForDisplayItems = ( cartData ) => {
```

Comment:
```suggestion
/**
 * Converts WC cart data into line items array for Stripe API.
 *
 * @param {Object} cartData WC cart data object.
 * @return {Array} Stripe API line items.
 */
export const transformCartDataForDisplayItems = ( cartData ) => {
```
Doc comment maybe? Same for the function below. 🙏